brainfuck/instructions.go

```diff
@@ -0,0 +1,116 @@
+package brainfuck
+
+import "fmt"
+
+//Block of constants which represents aliases for chars of
+//each command
+const (
+	Plus       byte = '+'
+	Minus      byte = '-'
+	RightShift byte = '>'
+	LeftShift  byte = '<'
+	WriteChar  byte = '.'
+	ReadChar   byte = ','
+	LoopStart  byte = '['
+	LoopEnd    byte = ']'
+)
+
+//sizeOfCell represents capacity of cell, you can modify it,
+//if you need
+const sizeOfCell = 1000
+
+//memoryCell struct represents structure and functionality
+//of real cell in memory
+type memoryCell struct {
+	cell [sizeOfCell]byte
+	ptr  int
+}
+
+//instruction interface implements compile()
+//Via this method we can addd new functionality to Brainfuck interpreter
+type instruction interface {
+	compile(m *memoryCell)
+}
+
+//plus type relates to char '+'
+type plus struct{}
+
+//implements compile for type plus
```

Comment:
no need to say that it **compiles** something. Say what it does exactly.
for example: `increments memory +1 ` or something like this

The same for the other commands

Reply:
fixed